upgrade.yml

```diff
@@ -416,6 +416,11 @@
   any_errors_fatal: true
   tags: swift
 
+  pre_tasks:
+    - name: stop swift services started by swift-init
+      command: swift-init stop all
+      ignore_errors: True
```

Comment:
If we stop all the swift services, all at once, wouldn't that put swift in a very broken state when it comes back?